src/runtime-rs/crates/hypervisor/src/dragonball/inner.rs

```diff
@@ -100,11 +103,22 @@ impl DragonballInner {
         let rootfs_driver = self.config.blockdev_info.block_device_driver.clone();
 
         // get kernel params
-        let mut kernel_params = KernelParams::new(self.config.debug_info.enable_debug);
+        let mut kernel_params = KernelParams::new();
         kernel_params.append(&mut KernelParams::new_rootfs_kernel_params(&rootfs_driver));
         kernel_params.append(&mut KernelParams::from_string(
             &self.config.boot_info.kernel_params,
         ));
+
+        if self.config.debug_info.enable_debug {
+            // export guest kernel log
+            kernel_params.push(Param::new("console", "ttyS1"));
```

Comment:
suggest adding const

Reply:
I have considered not to make them const value for the following reason:

- console and ttyS1 seems to be used only once
- there is no special meaning to this two string, if we add const, we just change the letter case

If you think that does not make sense, or we should always apply const for a const string, I will fix it.

Reply:
fair enough